internal/grpc/services/gateway/ocmshareprovider.go

```diff
@@ -388,22 +388,22 @@ func (s *svc) UpdateReceivedOCMShare(ctx context.Context, req *ocm.UpdateReceive
 							},
 						},
 					}
-					req := &datatx.PullTransferRequest{
+					req := &datatx.CreateTransferRequest{
 						SrcTargetUri:  srcTargetURI,
 						DestTargetUri: destTargetURI,
 						Opaque:        opaqueObj,
```

Comment:
As we're renaming those methods, what about cleaning the API as well? that is promoting this opaque info into a proper property of the `CreateTransferRequest` object?

Reply:
Share id is promoted to real property.